stories/components/AlignGroup.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/react';
+
+import { AlignGroup, AlignGroupProps } from '../../src';
+
+const DEBUG_COLOR = 'yellow';
+
+const control = { type: 'select', options: ['start', 'middle', 'end'] };
+export default {
+  title: 'Docs/Shapes',
+  component: AlignGroup,
+  argTypes: {
+    x: {
+      defaultValue: 100,
+      control: 'number',
+    },
+    y: {
+      defaultValue: 100,
+      control: 'number',
+    },
+    verticalAlign: {
+      defaultValue: 'start',
+      control,
+    },
+    horizontalAlign: {
+      defaultValue: 'start',
+      control,
+    },
+  },
```

Comment:
```suggestion
```

Replace with :

```
args: {
  x: 100,
  y: 100,
  ...
},
```

The types are handled automatically by storybook from the TypeScript interface.
I know we didn't do it in react-plot and it's because we didn't know exactly how it worked.